doc/release-process.md

```diff
@@ -37,6 +37,12 @@ Release Process
   - This update should be reviewed with a reindex-chainstate with assumevalid=0 to catch any defect
      that causes rejection of blocks in the past history.
 - Clear the release notes and move them to the wiki (see "Write the release notes" below).
+- Translations on Transifex
+    - Create [a new resource](https://www.transifex.com/bitcoin/bitcoin/content/) named after the major version with the slug `[bitcoin.qt-translation-<RRR>x]`, where `RRR` is the major branch number padded with zeros. Use `src/qt/locale/bitcoin_en.ts` to create it.
+    - Copy over the translations of the previous version using [transifex-migrate-resource](https://github.com/bitcoin-core/bitcoin-maintainer-tools#transifex-migrate-resource).
```

Comment:
I don't think the `transifex-migrate-resource.py` script in its current state will work for XLIFF translation files.
And Transifex "Translation Memory Fillup" pre-translation option should work for us.

It's definitely requires @laanwj's opinion.

Reply:
Considering the recent [translation update](https://github.com/bitcoin-core/gui/pull/377) incident, I'd suggest to replace this (outdated?) instruction with the following:

"Assure that in the project workflow setting (1) the ['Translation Memory Fillup'](https://docs.transifex.com/translation-memory/enabling-autofill) is _enabled_, and (2) ['Translation Memory Context Matching'](https://docs.transifex.com/translation-memory/translation-memory-with-context) is disabled".

Reply:
Done

Reply:
Thanks for bringing it up, I don't expect `transifex-migrate-resource.py` to ever work again, for two reasons:
- As @hebasto mentions, XLIFF probably requires a different treatment.
- The Transifex API version used by the script (v2) is deprecated and will be removed (Starting [April 7th, 2022](https://docs.transifex.com/api/introduction). apparently). But it was already working less well for recent releases.

So if it can be replaced by something Transifex provides themselves that would be nice.